cmd/skopeo/copy.go

```diff
@@ -24,7 +24,8 @@ type copyOptions struct {
 	signByFingerprint string          // Sign the image using a GPG key with the specified fingerprint
 	format            optionalString  // Force conversion of the image to a specified format
 	quiet             bool            // Suppress output information when copying images
-
+	srcNoCreds        bool
+	destNoCreds       bool
```

Comment:
This seems unused, along with all the other variables added outside of `utils.go`.